### PR TITLE
DOCS: Host python notebooks containing different data types using MyST-NB

### PR DESCRIPTION
**Pull request by derha**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

- [x] Issue reference: https://github.com/moja-global/moja_global_docs/issues/161
- [x] Subject Matter Expert (SME) reviewer: @aornugent @chicken-biryani 
- [x] Peer reviewer: @Namyalg 
- [x] Preview Links: https://moja-global-documentation--178.org.readthedocs.build/en/178/Understanding-FLINT/FLINT-Inputs/index.html
- [x] Explain the changes: The Jupyter Notebooks made by @padmajabhol that investigate FLINT data types were parsed and hosted on documentation website.
- [x] Dependency changes: Yes, sphinx MyST-NB extension for compiling Jupyter Notebooks was used.

Thanks for opening the Pull Request for moja global. Happy contributing ✨

### CODE REVIEW

**Comment by padmajabhol** [comment]
There are a few issues with the notebooks from my end. Will correct them. All the checks have passed,  thanks a lot @derha !

**Comment by Namyalg** [comment]
Yes, as the pandoc issue is resolved, we can go ahead with this

**Comment by derha** [comment]
I'm glad I could help!

File: docs/Understanding-FLINT/FLINT-Inputs/index.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FLINT-UI Inputs
+=========================
+
+We currently use five main data-types that are used as FLINT inputs - geoTIFF, JSON, CSV, SQL DB, Shapefile. We have hosted the python notebooks for each data type that contains the instructions and the required dependencies to view each file on a python notebook.
```
Comment (padmajabhol) [inline docs/Understanding-FLINT/FLINT-Inputs/index.rst:4]: Replace line 4 with : We currently have five main data types used as FLINT inputs - geoTIFF, JSON, CSV, SQL DB, and Shapefile. Final change and the PR is good to go! 🚀

**Review by padmajabhol — CHANGES_REQUESTED** [review]

**Review by padmajabhol — APPROVED** [review]

**Comment by aornugent** [comment]
Thanks @derha!

@padmajabhol, I think you have permissions to merge in docs repositories?  🤞

**Comment by padmajabhol** [comment]
> Thanks @derha!
> 
> @padmajabhol, I think you have permissions to merge in docs repositories? 🤞

Yes!

**Pull request merged**